definitions/ext-router/mikrotik-dashboard.json

```diff
@@ -0,0 +1,181 @@
+{
```

Comment:
The PR is failing because of the dashboard, you can fix it by [sanitizing it](https://github.com/newrelic-experimental/entity-synthesis-definitions#dashboard-templates) with:

```
docker-compose run sanitize-dashboards
```

Reply:
Done! 
I didn't realize that I could not export a NR dashboard as a json and paste it here out of the box

Reply:
Thanks! sorry for that 🙇🏻‍♀️